be/src/column/array_column.h

```diff
@@ -221,6 +221,10 @@ class ArrayColumn final : public ColumnFactory<Column, ArrayColumn> {
 
     std::string debug_string() const override;
 
+    bool exceed_capacity_limit() const override {
```

Comment:
Mentioned casually, capacity has the limit meaning somewhat already, but some article use limit as well, not necessary to change as far as I can see.